syntax = "proto3";

package mprpc;

message RpcHeader{
    bytes service_name = 1;
    bytes service_method = 2;
    uint32 arg_size = 3;
}